kpi/deployment_backends/kobocat_backend.py

```diff
@@ -11,16 +11,18 @@
 from datetime import date, datetime
 from typing import Generator, Optional, Union
 from urllib.parse import urlparse
-from xml.etree import ElementTree as ET
+from xml.etree.ElementTree import ElementTree, ParseError
 try:
     from zoneinfo import ZoneInfo
 except ImportError:
     from backports.zoneinfo import ZoneInfo
 
 import requests
+from defusedxml.ElementTree import fromstring, tostring
+from defusedxml.lxml import tostring
```

Comment:
Hi @JacquelineMorrissette, could you double-check this PR with a few things in mind?
1. `fromstring` and `tostring` are awfully generic-sounding things. I think the usual convention of `… import ElementTree as ET`, an then using stuff like `ET.fromstring` is good to avoid ambiguity.
    1. Watch out for shadowing one import with another, as in this case where the second `tostring` import completely replaces the first.
1. Why are we still using `xml.etree.ElementTree` and `lxml` versus `defusedxml.ElementTree` and `defusedxml.lxml`? Are there certain things we need that the defusedxml alternatives don't provide?

Edit: whoops, I just saw this in defusedxml's readme:
> defusedxml.lxml
> DEPRECATED The module is deprecated and will be removed in a future release.

Given that, please don't use `defusedxml.lxml` anywhere. Try to see how our uses of `lxml` could be replaced with `ElementTree`, and don't hesitate to ask questions :)

Reply:
1. Will do, I'll give it another pass
2. The reason I kept some uses of `xml.etree.ElementTree` (and others) is because the defusedxml library isn't a complete replacement for most features of these and only replaces specific functionalities such as `tostring` and `fromstring`. For example: defusexml doesn't have an equivalent to `from xml.etree.ElementTree import ElementTree`
The documentation here lists what it replaces, in case it helps: https://pypi.org/project/defusedxml/#defusedxml-elementtree